### PR TITLE
Add `query-params` Validation

### PR DESCRIPTION
**Pull request by allentiak**

### CODE REVIEW

File: modules/operation-measure-evaluate-measure/Makefile [hunk 1]
```diff
@@ -13,6 +13,12 @@ test: prep
 test-coverage: prep
 	clojure -M:test:coverage
 
+test-watch: prep
+	clojure -X:test:kaocha-watch --profile :watch
+
+repl: prep
```
Comment (alexanderkiel) [inline modules/operation-measure-evaluate-measure/Makefile:19]: should be called emacs-repl

File: modules/operation-measure-evaluate-measure/deps.edn [hunk 2]
```diff
@@ -34,13 +34,33 @@
    {blaze/db-stub
     {:local/root "../db-stub"}}}
 
+  ;; For both toying at the terminal and editor support (via nREPL)
+  ;; Supports Emacs' refactor-nrepl and sayid
+  :repl
```
Comment (alexanderkiel) [inline modules/operation-measure-evaluate-measure/deps.edn:39]: should be called emacs-repl

File: modules/operation-measure-evaluate-measure/deps.edn [hunk 3]
```diff
@@ -34,13 +34,33 @@
...
+  {:extra-deps {nrepl/nrepl                   {:mvn/version "1.3.0"}
```
Comment (alexanderkiel) [inline modules/operation-measure-evaluate-measure/deps.edn:40]: please use the indentation we discussed in the global deps.edn

File: modules/operation-measure-evaluate-measure/deps.edn [hunk 4]
```diff
@@ -34,13 +34,33 @@
...
    {lambdaisland/kaocha
     {:mvn/version "1.91.1392"}}
 
    :main-opts ["-m" "kaocha.runner"]}
 
+  :kaocha-watch
+  {:extra-deps
+   {lambdaisland/kaocha
+    {:mvn/version "1.91.1392"}}
+   :exec-fn   kaocha.runner/exec-fn
```
Comment (alexanderkiel) [inline modules/operation-measure-evaluate-measure/deps.edn:59]: I don't know about this new kind of execution config. But if that is now preferred over `:main-opts` we may use it everywhere?

File: modules/operation-measure-evaluate-measure/tests.edn [hunk 5]
```diff
@@ -1,11 +1,13 @@
 #kaocha/v1
  #merge
   [{}
-   #profile {:ci {:reporter kaocha.report/documentation
-                  :color? false}
+   #profile {:ci       {:reporter kaocha.report/documentation
```
Comment (alexanderkiel) [inline modules/operation-measure-evaluate-measure/tests.edn:4]: please don't align map values

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

File: modules/operation-measure-evaluate-measure/deps.edn (hunk 4 above)
Comment (allentiak) [reply modules/operation-measure-evaluate-measure/deps.edn:59]: I certainly agree in your consistency argument. That being said, I am not sure whether there is a preferred way at all or not - much less which one that would be.

So, I will leave this one open for now so I can research it further later on.

Comment (allentiak) [reply modules/operation-measure-evaluate-measure/deps.edn]: For now, I will leave this change outside of this PR.
I will review this later.

Comment (allentiak) [reply modules/operation-measure-evaluate-measure/deps.edn]: See https://github.com/samply/blaze/issues/2105

**Review by alexanderkiel — APPROVED** [review]

**Pull request merged**